airsonic-main/src/main/java/org/airsonic/player/spring/ServletConfiguration.java

```diff
@@ -0,0 +1,63 @@
+package org.airsonic.player.spring;
+
+import org.airsonic.player.controller.PodcastController;
+import org.airsonic.player.service.SecurityService;
+import org.airsonic.player.service.SettingsService;
+import org.airsonic.player.theme.CustomThemeResolver;
+import org.airsonic.player.theme.CustomThemeSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.web.servlet.ThemeResolver;
+import org.springframework.web.servlet.ViewResolver;
+import org.springframework.web.servlet.handler.SimpleUrlHandlerMapping;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+import java.util.Properties;
+
+@Configuration
+public class ServletConfiguration {
+
+    @Bean
+    public SimpleUrlHandlerMapping urlMapping(PodcastController podcastController) {
+        SimpleUrlHandlerMapping handlerMapping = new SimpleUrlHandlerMapping();
+        handlerMapping.setAlwaysUseFullPath(true);
+        Properties properties = new Properties();
+        properties.put("/podcast/**", podcastController);
+        handlerMapping.setMappings(properties);
+        return handlerMapping;
+    }
+
+    @Bean
+    public ResourceBundleMessageSource messageSource() {
```

Comment:
This may be autoinstantiated by putting in a property in the application.properties:
```
spring.messages.basename=org.airsonic.player.i18n.ResourceBundle
```
Do not need a bean here.

Reply:
Let's change this in a separate PR.